src/Button/index.js

```diff
@@ -1,365 +1,332 @@
+        success: theme.colorTextButtonSuccessHover,
+        warning: theme.colorTextButtonWarningHover,
+        critical: theme.colorTextButtonCriticalHover,
+        facebook: theme.colorTextButtonFacebookHover,
+        google: theme.colorTextButtonGoogleHover,
+      },
+      bordered: {
+        primary: theme.colorTextButtonPrimaryBorderedHover,
+        secondary: theme.colorTextButtonSecondaryBorderedHover,
+        info: theme.colorTextButtonInfoBorderedHover,
+        success: theme.colorTextButtonSuccessBorderedHover,
+        warning: theme.colorTextButtonWarningBorderedHover,
+        critical: theme.colorTextButtonCriticalBorderedHover,
+        facebook: theme.colorTextButtonFacebookBorderedHover,
+        google: theme.colorTextButtonGoogleBorderedHover,
+      },
+    },
+    colorTextButtonActive: {
+      filled: {
+        primary: theme.colorTextButtonPrimaryActive,
+        secondary: theme.colorTextButtonSecondaryActive,
+        info: theme.colorTextButtonInfoActive,
+        success: theme.colorTextButtonSuccessActive,
+        warning: theme.colorTextButtonWarningActive,
+        critical: theme.colorTextButtonCriticalActive,
+        facebook: theme.colorTextButtonFacebookActive,
+        google: theme.colorTextButtonGoogleActive,
+      },
+      bordered: {
+        primary: theme.colorTextButtonPrimaryBorderedActive,
+        secondary: theme.colorTextButtonSecondaryBorderedActive,
+        info: theme.colorTextButtonInfoBorderedActive,
+        success: theme.colorTextButtonSuccessBorderedActive,
+        warning: theme.colorTextButtonWarningBorderedActive,
+        critical: theme.colorTextButtonCriticalBorderedActive,
+        facebook: theme.colorTextButtonFacebookBorderedActive,
+        google: theme.colorTextButtonGoogleBorderedActive,
+      },
+    },
+    borderColorButton: {
+      bordered: {
+        primary: theme.borderColorButtonPrimaryBordered,
+        secondary: theme.borderColorButtonSecondaryBordered,
+        info: theme.borderColorButtonInfoBordered,
+        success: theme.borderColorButtonSuccessBordered,
+        warning: theme.borderColorButtonWarningBordered,
+        critical: theme.borderColorButtonCriticalBordered,
+        facebook: theme.borderColorButtonFacebookBordered,
+        google: theme.borderColorButtonGoogleBordered,
+      },
+    },
+    borderColorButtonHover: {
+      bordered: {
+        primary: theme.borderColorButtonPrimaryBorderedHover,
+        secondary: theme.borderColorButtonSecondaryBorderedHover,
+        info: theme.borderColorButtonInfoBorderedHover,
+        success: theme.borderColorButtonSuccessBorderedHover,
+        warning: theme.borderColorButtonWarningBorderedHover,
+        critical: theme.borderColorButtonCriticalBorderedHover,
+        facebook: theme.borderColorButtonFacebookBorderedHover,
+        google: theme.borderColorButtonGoogleBorderedHover,
+      },
+    },
+    borderColorButtonActive: {
+      bordered: {
```

Comment:
I think it shouldn't be nested right ?

Reply:
This `tokens` concept is from the old version where was `variation` as prop. I can rewrite this as it was in the very first version, e.g.:
`color: ${ bordered ? colorBordered : color }`